packages/navstar/build

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -x
+export LD_LIBRARY_PATH=/opt/mesosphere/lib:/opt/mesosphere/active/libsodium/lib
+export CFLAGS="-I/opt/mesosphere/include -I/opt/mesosphere/active/libsodium/include"
```

Comment:
Everything in "/lib" inside libsodium should end up at /opt/mesosphere/lib, so this and he lib line shouldn't be neededm